apps/protocol-frontend/src/components/ContributionMembersPie.tsx

```diff
@@ -21,7 +21,13 @@ const ContributionMembersPie = ({
   isLoading,
   isError,
 }: ContributionMembersPieProps) => {
-  const isMobile = useBreakpointValue({ base: true, lg: false });
+  const isSmallerScreen = useBreakpointValue({
+    base: true,
+    md: true,
+    lg: true,
+    xl: false,
+    '2xl': false,
+  });
```

Comment:
When testing this seems to make the dashboard view less responsive and cuts off some data

Reply:
Should be good now -- I used a separate one for the widths than the layers